www/javascript/site-jw-player-media-display.js

```diff
@@ -560,22 +560,24 @@ SiteJwPlayerMediaDisplay.prototype.drawDialogs = function()
 	this.overlay.appendChild(document.createTextNode(''));
 	this.container.appendChild(this.overlay);
 
-	// if the video has been watched before, and we're more than 60s from the
-	// end or start, show a message allowing the viewed to resume or start
-	// from the beginning
-	if (this.start_position > 0) {
+	 // if the video has been watched before, and we're more than
+	 // 10s from the start, or 60s from the end, show a
+	 // message allowing the viewed to resume or start
+	 // from the beginning
+	 if (this.start_position > 0) {
```

Comment:
This block of lines has a space after the tab that should be removed.

Reply:
Fixed the leading spaces.